common/os.go

```diff
@@ -18,7 +17,7 @@ var (
 func TrapSignal(cb func()) {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt)
-	signal.Notify(c, syscall.SIGTERM)
+	signal.Notify(c, os.Kill) // nolint: megacheck
```

Comment:
turns out we do need to change this - os.Kill can't be caught!

https://github.com/dominikh/go-staticcheck/issues/26